src/content/docs/setup/analytics/instrumentation.mdx

```diff
@@ -1,58 +1,205 @@
+
+### Required Configuration Parameters
+
+<CodeExample code={`
+ ...
+  "analytics": {
+    "base-currency-code": "USD",
+    "environment": "Production",
+    "store-id": 1,
+    "store-name": "Main Website Store",
+    "store-url": "{{YOUR_STORE_URL}}",
+    "store-view-id": 1,
+    "store-view-name": "Default Store View",
+    "website-id": 1,
+    "website-name": "Main Website"
+  }
+  ...
+`} lang="json" title="config.json" />
+
+### Configuration Parameters Explained
+
+| Parameter | Description | Example | Required |
+|-----------|-------------|---------|----------|
+| `base-currency-code` | Default currency for your store | `"USD"`, `"EUR"` | Yes |
+| `environment` | Current environment | `"Production"`, `"Staging"` | Yes |
+| `store-id` | Unique identifier for your store | `1`, `2` | Yes |
+| `store-name` | Human-readable store name | `"Main Website Store"` | Yes |
+| `store-url` | Your store's primary URL | `"https://example.com"` | Yes |
+| `store-view-id` | Unique identifier for store view | `1`, `2` | Yes |
+| `store-view-name` | Human-readable store view name | `"Default Store View"` | Yes |
+| `website-id` | Unique identifier for website | `1`, `2` | Yes |
+| `website-name` | Human-readable website name | `"Main Website"` | Yes |
+
+### Data Services Configuration
+
+For the instrumentation to work with Adobe Commerce's data services, you'll need additional configuration parameters. The easiest way to obtain these is through the `magento/module-data-services-graphql` module, which exposes the necessary GraphQL endpoints.
+
+#### Required for Data Services
+
+- **Catalog Service credentials**: For product data synchronization
+- **SaaS environment ID**: Links your storefront to Adobe Commerce SaaS services
+- **API keys**: Authenticate with Adobe Commerce backend services
+
+## Event Collection and Validation
+
+### Automatic Event Collection
+
+The Commerce boilerplate includes the [Storefront Events Collector](https://github.com/adobe/commerce-events/tree/main/packages/storefront-events-collector), which automatically:
+
+<Steps>
+1. **Listens for ACDL events**: Monitors the data layer for new events
+1. **Validates event structure**: Ensures events conform to required schemas  
+1. **Batches and sends data**: Efficiently transmits events to Adobe Commerce
+1. **Handles errors**: Manages network issues and retry logic
+</Steps>
+
+### Event Types Collected
+
+Your instrumentation will automatically track:
+
+- **Product interactions**: Views, clicks, add-to-cart, remove-from-cart
+- **Search behavior**: Search queries, results clicked, filters applied
+- **Shopping flow**: Checkout initiation, payment method selection, purchase completion
+- **Page navigation**: Page views, time on page, bounce events
```

Comment:
same comment as for src/content/docs/setup/analytics/adobe-experience-platform.mdx

Reply:
wait, my comment in src/content/docs/setup/analytics/adobe-experience-platform.mdx was about AEP events.
Are we talking here about Live Search events? then we have a little bit different set of events we send to snowplow